src/Autocomplete.js

```diff
@@ -55,6 +63,37 @@ class Autocomplete extends AutocompleteBase {
 	}
 
 	/**
+	 * Actives an option of the suggestion list by inform an index.
+	 * @param {number} index
+	 * @protected
+	 */
+	activeListItem_(index) {
+		let option = this.currentList_[index];
+		dom.removeClasses(this.currentList_[this.activedIndex_], 'active');
+		this.activedIndex_ = index;
```

Comment:
I had commented this the last time, but I've noticed that github showed this and another comment I'd made as "outdated" to you, which is why you probably didn't notice them. So I'm copying them here again:

The correct in English would be either `activatedIndex_` or `activeIndex_`, not `activedIndex_`, can you do this small fix? Same for `activeListItem_`, it should be `activateListItem_`.